README.md

```diff
@@ -1,20 +1,35 @@
 ### dependency-tree [![npm](http://img.shields.io/npm/v/dependency-tree.svg)](https://npmjs.org/package/dependency-tree) [![npm](http://img.shields.io/npm/dm/dependency-tree.svg)](https://npmjs.org/package/dependency-tree)
 
-> Get the dependency tree of a module (as a list)
+> Get the dependency tree of a module
 
 `npm install dependency-tree`
 
 ### Usage
 
 ```js
-var getTreeAsList = require('dependency-tree');
+var dependencyTree = require('dependency-tree');
 
-// Returns a list of filepaths for all visited dependencies
-var tree = getTreeAsList('path/to/a/file', 'path/to/all/js/files');
+// Returns a dependency tree for the given file
+var tree = dependencyTree('path/to/a/file', 'path/to/all/js/files');
+
+// Returns a pre-order traversal of the tree with duplicate sub-trees pruned.
+var preOrderList = dependencyTree.traversePreOrder(tree);
```

Comment:
I had no real intention behind doing a pre-order traversal. Perhaps all of this could be masked behind `dependencyTree.asList(tree)` that returns a postOrder list.